tests/integration-tests/tests/dashboard/test_dashboard.py

```diff
@@ -31,17 +31,38 @@ def test_dashboard(
     )
     cluster = clusters_factory(cluster_config)
     cw_client = boto3.client("cloudwatch", region_name=region)
-    dashboard_name = "{0}-{1}".format(cluster.cfn_name, region)
 
+    # test CWAgent metrics
+    metrics_response = cw_client.list_metrics(Namespace="CWAgent")
+    metrics = [metric["MetricName"] for metric in metrics_response["Metrics"]]
+    assert_that(metrics).contains("mem_used_percent")
+    assert_that(metrics).contains("disk_used_percent")
+
+    # test dashboard and alarms
+    dashboard_name = "{0}-{1}".format(cluster.cfn_name, region)
     if dashboard_enabled:
-        response = cw_client.get_dashboard(DashboardName=dashboard_name)
-        assert_that(response["DashboardName"]).is_equal_to(dashboard_name)
+        # test dashboard
+        dashboard_response = cw_client.get_dashboard(DashboardName=dashboard_name)
+        assert_that(dashboard_response["DashboardName"]).is_equal_to(dashboard_name)
         if cw_log_enabled:
-            assert_that(response["DashboardBody"]).contains("Head Node Logs")
+            assert_that(dashboard_response["DashboardBody"]).contains("Head Node Logs")
         else:
-            assert_that(response["DashboardBody"]).does_not_contain("Head Node Logs")
+            assert_that(dashboard_response["DashboardBody"]).does_not_contain("Head Node Logs")
+        # test alarms
+        mem_alarm = f"{cluster.cfn_name}_MemAlarm_HeadNode"
+        disk_alarm = f"{cluster.cfn_name}_DiskAlarm_HeadNode"
+        alarm_response = cw_client.describe_alarms(AlarmNamePrefix=cluster.cfn_name)
+        alarms = [alarm["AlarmName"] for alarm in alarm_response["MetricAlarms"]]
```

Comment:
Apart from checking alarm names, we can also check the threshold, dimension, static, period that are in the output of describe_alarms

Reply:
sounds good